resources/services/glue/dev_endpoints.go

```diff
@@ -0,0 +1,229 @@
+				Description: "Glue version determines the versions of Apache Spark and Python that Glue supports",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "last_modified_timestamp",
+				Description: "The point in time at which this DevEndpoint was last modified",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "last_update_status",
+				Description: "The status of the last update",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "number_of_nodes",
+				Description: "The number of Glue Data Processing Units (DPUs) allocated to this DevEndpoint",
+				Type:        schema.TypeBigInt,
+			},
+			{
+				Name:        "number_of_workers",
+				Description: "The number of workers of a defined workerType that are allocated to the development endpoint",
+				Type:        schema.TypeBigInt,
+			},
+			{
+				Name:        "private_address",
+				Description: "A private IP address to access the DevEndpoint within a VPC if the DevEndpoint is created within one",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "public_address",
+				Description: "The public IP address used by this DevEndpoint",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "public_key",
+				Description: "The public key to be used by this DevEndpoint for authentication",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "public_keys",
+				Description: "A list of public keys to be used by the DevEndpoints for authentication",
+				Type:        schema.TypeStringArray,
+			},
+			{
+				Name:        "role_arn",
+				Description: "The Amazon Resource Name (ARN) of the IAM role used in this DevEndpoint",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "security_configuration",
+				Description: "The name of the SecurityConfiguration structure to be used with this DevEndpoint",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "security_group_ids",
+				Description: "A list of security group identifiers used in this DevEndpoint",
+				Type:        schema.TypeStringArray,
+			},
+			{
+				Name:        "status",
+				Description: "The current status of this DevEndpoint",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "subnet_id",
+				Description: "The subnet ID for this DevEndpoint",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "vpc_id",
+				Description: "The ID of the virtual private cloud (VPC) used by this DevEndpoint",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "worker_type",
+				Description: "The type of predefined worker that is allocated to the development endpoint Accepts a value of Standard, G1X, or G2X  * For the Standard worker type, each worker provides 4 vCPU, 16 GB of memory and a 50GB disk, and 2 executors per worker  * For the G1X worker type, each worker maps to 1 DPU (4 vCPU, 16 GB of memory, 64 GB disk), and provides 1 executor per worker",
```

Comment:
I guess we may want to trim this description a little bit, removing everything from the first `*` onwards.